config/zones.json

```diff
@@ -115,6 +115,16 @@
       "production": "BG.fetch_production"
     }
   },
+  "BO": {
+    "capacity": {
+      "hydro": 483.22,
+      "unknown" : 1344.55,
+      "wind": 27 
+    },
+    "parsers": {
+      "production": "BO.fetch_production"
```

Comment:
can you also add the generation forecast as you have it in the parser? There's an example for DK.